langs/nim/Dockerfile

```diff
@@ -14,21 +14,17 @@ RUN ./build.sh             \
  && ./install.sh /         \
  && strip /nim/bin/nim
 
-# Docker can only copy symlinks by copying the whole folder.
-RUN mkdir /usr-lib-symlinks \
- && cp -P /usr/lib/libc.so /usr-lib-symlinks
-
```

Comment:
How come this was removed? The entire point of only copying the symlink was to avoid two copies of musl in the final image.